packages/salesforcedx-vscode-apex/package.json

```diff
@@ -92,16 +92,23 @@
       "view/item/context": [
         {
           "command": "sfdx.force.test.view.showError",
-          "when": "view == sfdx.force.test.view"
+          "when":
+            "view == sfdx.force.test.view && viewItem =~ /(apexTest|apexTestGroup)_Fail/"
+        },
+        {
+          "command": "sfdx.force.test.view.showDefinition",
+          "when":
+            "view == sfdx.force.test.view && viewItem =~ /(apexTest|apexTestGroup)(_Pass|_Skip|\\b)/"
```

Comment:
VSCode doesn't exactly have a way of dynamically updating the labels for the same command in a context menu. I created another command that runs the same method, but has the "Go to Definition" label. Then conditionally render one command or the other based on the where clause.

Reply:
1. Yeah, there isn't a way to dynamically update the labels. 
1. Does this mean that you won't be able to see the "Go to Definition" menu if you are on a failing node?
1. What about calling it `goToDefinition`. I think you're really going to the definition more than showing it in-line.